test/k8sT/Policies.go

```diff
@@ -263,11 +263,6 @@ var _ = SkipDescribeIf(func() bool {
 				namespaceForTest, appPods[helpers.App2], namespaceForTest, appPods[helpers.App1]))
 			trace.ExpectContains("Final verdict: ALLOWED", "Policy trace output mismatch")
 
-			trace = kubectl.CiliumExecMustSucceed(context.TODO(), ciliumPod, fmt.Sprintf(
-				"cilium policy trace --src-k8s-pod %s:%s --dst-k8s-pod %s:%s",
-				namespaceForTest, appPods[helpers.App3], namespaceForTest, appPods[helpers.App1]))
-			trace.ExpectContains("Final verdict: DENIED", "Policy trace output mismatch")
-
```

Comment:
> If you see the test cases, we have 2 categories as follows
> 
> 1. with `--dport `argument test cases
> 2. without `--dport `argument test cases
> 
> As we made `--dport ` as a mandatory argument, so we removed the rest of the void (unused `--dport` option) test cases.
> And we are already taking care of `--dport` argument test cases so no need for any changes

@Maddy007-maha Thanks for your answer. I agree with the fact that we could have removed the tests without `--dport` if they were duplicated with the `--dport` tests, but does not seem to be the case to me.

Let's take a look at this example. We have two tests:

1. With `--dport`, using `helpers.App2`, expecting `ALLOWED`:

```go
trace := kubectl.CiliumExecMustSucceed(context.TODO(), ciliumPod, fmt.Sprintf(
		"cilium policy trace --src-k8s-pod %s:%s --dst-k8s-pod %s:%s --dport 80/TCP",
	namespaceForTest, appPods[helpers.App2], namespaceForTest, appPods[helpers.App1]))
trace.ExpectContains("Final verdict: ALLOWED", "Policy trace output mismatch")
```

2. Without `--dport`, using `helpers.App3`, expecting `DENIED`:

```
trace = kubectl.CiliumExecMustSucceed(context.TODO(), ciliumPod, fmt.Sprintf(
	"cilium policy trace --src-k8s-pod %s:%s --dst-k8s-pod %s:%s",
	namespaceForTest, appPods[helpers.App3], namespaceForTest, appPods[helpers.App1]))
trace.ExpectContains("Final verdict: DENIED", "Policy trace output mismatch")
```

They look similar, but are not duplicates: my understanding is that they work hand-in-hand, and both are required to cover what we want to test. Thus we can't simply remove them: they should be updated to use `--dport` instead :)